app/pages/task-list/helpers/generateTaskList.test.js

```diff
@@ -0,0 +1,110 @@
+import { generateTaskList } from './generateTaskList';
+import { baseUrl } from '../../../config';
+
+const orderId = 'neworder';
+
+const taskListManifest = {
+  tasks: [{
+    name: 'task 1',
+    sections: [{
+      id: 'task1item1',
+      title: 'task 1 item 1',
+    }],
+  }, {
+    name: 'task 2',
+    sections: [{
+      id: 'task2item1',
+      title: 'task 1 item 1',
+      dependencies: ['task1item1'],
+    }, {
+      id: 'task2item2',
+      title: 'task 1 item 2',
+      dependencies: ['task1item1', 'task2item1'],
+    }],
+  }],
+};
+
+const sectionsData = [
+  {
+    id: 'task1item1',
+    status: 'complete',
+  },
+  {
+    id: 'task2item1',
+    status: 'complete',
+  },
+  {
+    id: 'task2item2',
+    status: 'incomplete',
+  },
+];
+// [
+//   { taskName: 'task 1', items: [ [Object] ] },
+//   { taskName: 'task 2', items: [ [Object], [Object] ] }
+// ]
+describe('generateTaskList', () => {
+  describe('taskName', () => {
+    it('should return task.name as taskName in tasks array for each task in manifest', () => {
```

Comment:
I find this test description a little confusing.  Can you change it please

Reply:
yeah me too 😄

Reply:
I didn't even know what to suggest here 😂